frb/builds/build_hosts.py

```diff
@@ -1760,41 +2085,836 @@ def build_host_200430(build_ppxf=False, build_photom=False, build_cigale=False,
+    #photom_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 
+    #                           'bhandari2021_photom_liz.ascii')
+    if build_photom:
+        # PS1
+        #search_r = 1 * units.arcsec
+        #ps1_srvy = panstarrs.Pan_STARRS_Survey(gal_coord, search_r)
+        #ps1_tbl = ps1_srvy.get_catalog(print_query=True)
+        #ps1_tbl['Name'] = host200906.name
+
+        #Panstarrs
+        photom = Table()
+        photom['Name'] = ['HG{}'.format(frbname)]
+        photom['ra'] = host171020.coord.ra.value
+        photom['dec'] = host171020.coord.dec.value
+        photom['Pan-STARRS_g'] = 15.489
+        photom['Pan-STARRS_g_err'] = 0.009
+        photom['Pan-STARRS_r'] = 15.36
+        photom['Pan-STARRS_r_err'] = 0.02
+        photom['Pan-STARRS_i'] = 15.28 
+        photom['Pan-STARRS_i_err'] = 0.01
+        photom['Pan-STARRS_z'] = 15.361
+        photom['Pan-STARRS_z_err'] = 0.005
+        photom['Pan-STARRS_y'] = 15.41
+        photom['Pan-STARRS_y_err'] = 0.02
+        photom['WISE_W1'] = 14.667
+        photom['WISE_W1_err'] = 0.037
+        photom['WISE_W2'] = 14.581
+        photom['WISE_W2_err'] = 0.066
+        photom['WISE_W3'] = 11.541
+        photom['WISE_W3_err'] = 0.234
+        photom['WISE_W4'] = 8.395   
+        photom['WISE_W4_err']= 0.435
+        photom['VISTA_Y'] = 14.53
+        photom['VISTA_Y_err'] = 0.0079
+        photom['VISTA_J'] = 14.53
+        photom['VISTA_J_err'] = 0.0142
+        photom['VISTA_Ks'] = 14.53
+        photom['VISTA_Ks_err'] = 0.0342
+
+        #Merge and write
+        photom = frbphotom.merge_photom_tables(photom, photom_file)
+        photom.write(photom_file, format=frbphotom.table_format, overwrite=True)
+        print("Wrote photometry to: {}".format(photom_file))
+    # Load
+    #photom = Table.read(photom_file, format=frbphotom.table_format)
+    # Dust correct
+    #EBV = nebular.get_ebv(gal_coord)['meanValue']  # 0.061
+    #frbphotom.correct_photom_table(photom, EBV, 'HG171020')
+    # Parse
+    #host171020.parse_photom(photom, EBV=EBV)
+    #print(host200906.photom.keys())
+    
+    # CIGALE
+    #cigale_file = os.path.join(db_path, 'Realfast', 'Bhandari2021', 'HG171020_CIGALE.fits')
+    #sfh_file = cigale_file.replace('CIGALE', 'CIGALE_SFH')
+    if build_cigale:
+        # Prep
+        #cut_photom = Table()
+        # Let's stick to DES only
+        #for key in host200906.photom.keys():
+         #   if 'DES' not in key:
+         #       continue
+         #   cut_photom[key] = [host200906.photom[key]]
+        # Run
+        #print(cut_photom)
+        print("Running cigale")
+        #cut_photom = None
+        # Run
+        print(host171020.z)
+        cigale.host_run(host171020,cigale_file=cigale_file) #, cut_photom=cut_photom, cigale_file=cigale_file)
+    # Parse
+    #warnings.warn("TURN 200906 CIGALE back on")
+    #host171020.parse_cigale(cigale_file, sfh_file=sfh_file)
+
+    # Derived quantities from Mahony2018
```

Comment:
As above, are these from CIGALE?

Reply:
Removed CIGALE bit as these are NOT derived from CIGALE.